lib/src/test_signing_backend.rs

```diff
@@ -63,13 +63,13 @@ impl SigningBackend for TestSigningBackend {
             Ok(Verification {
                 status: SigStatus::Good,
                 key,
-                display: None,
+                display: Some(self.name().into()),
```

Comment:
nit: maybe better to hardcode "test" here because `display` is NOT the backend name. (applies to the other changes)